docs/guides/cloud/users.mdx

```diff
@@ -54,6 +54,11 @@ Cypress Cloud. From there, you will have the choice of authenitication type:
 Refer to our
 [Enterprise SSO Guide](https://on.cypress.io/organizations#Slack-Integration)
 
+#### **Locked Out or Forgot Password**
```

Comment:
These section headers render as h4's so they don't need the additional "**" bold syntax. EAxtra high fives if you can pull them off of the previous headers as well while in here 🙏 😄